src/main/java/com/cargohub/models/RegistrationModel.java

```diff
@@ -0,0 +1,22 @@
+package com.cargohub.models;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RegistrationModel {
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    private String address;
+    private String phoneNumber;
+
+    private String cardNumber;
+    private String nameOnCard;
+    private String expirationMonth;
+    private String expirationYear;
+    private String billingAddress;
```

Comment:
better to move this properties to a separate model and include this model as a field of RegistrationModel. After such manipulation you can get rid of of mapping DTO-objects to each other in AuthenticationController in register method (*)